packages/suite/src/views/password-manager/PasswordManager/PasswordManager.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '@trezor/components';
+
+import { SectionItem, TextColumn, ActionColumn } from 'src/components/suite';
+import { useDevice, usePasswords } from 'src/hooks/suite';
+import { getNextId } from 'src/utils/suite/passwords';
+
+import { EntryForm } from './EntryForm';
+import { PasswordsList } from './PasswordsList';
+import { TagsList } from './TagsList';
+
+const PasswordManagerBody = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+`;
+
+export const PasswordManager = () => {
+    const {
+        entries,
+        tags,
+        entriesByTag,
+        isSomeTagSelected,
+        config,
+        fileName,
+        selectedTags,
+        setSelectedTags,
+        connect,
+        disconnect,
+        selectedProvider,
+        providerConnecting,
+        savePasswords,
+        device,
+    } = usePasswords();
+
+    const { isLocked } = useDevice();
+    const isDeviceLocked = isLocked();
+
+    const [formActive, setFormActive] = useState<null | number>(null);
+
+    if (!device?.state) {
```

Comment:
I wonder why this functionality is dependent on `device.state`? This way, you cannot access password manager before discovery.

Reply:
it could be reworked to show something, but it is not possible to have a full functionality without an authorized device